src/styles/Task.scss

```diff
@@ -24,6 +25,8 @@
 .task-content {
   align-self: center;
   background-color: #e0edf8;
+  padding-bottom: 1%;
```

Comment:
Why do we need to make this a percent? 1% seems very small - what happens if we do 0% padding?

Reply:
Without padding:
<img width="1320" alt="Screen Shot 2022-10-08 at 5 22 53 PM" src="https://user-images.githubusercontent.com/13056466/194732045-1186e267-eb7e-40f6-b251-c08a195455b2.png">
With padding:
<img width="1305" alt="Screen Shot 2022-10-08 at 5 22 42 PM" src="https://user-images.githubusercontent.com/13056466/194732041-76e49fde-9099-4a3f-b5eb-4525f1f9b49a.png">
Figma example:
<img width="662" alt="Screen Shot 2022-10-08 at 5 24 03 PM" src="https://user-images.githubusercontent.com/13056466/194732067-1f0ab739-bbcc-4149-ab97-5873a19e0277.png">

It matches the Figma design better